docs/community/cheat-sheets.md

```diff
@@ -0,0 +1,17 @@
+# Cheat Sheets
+
+Cheat sheets provide easy to use code snippets for common scenarios.
+
+> [!NOTE]
+> The below cheat sheets provide quick samples for the V8 API.
+
+## Download
+
+| Strategy | Pdf link |
+| ------------- | :-------------: |
+| Timeout | [Download](../media/Timeout%20cheat%20sheet.pdf) |
```

Comment:
Can you swap the spaces for dashes in the file names? Then we don't need to worry about encoding or quoting paths.